src/Model/Fulfilment/OrderLine.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Model\Fulfilment;
+
+use MyParcelNL\Sdk\src\Model\BaseModel;
+
+class OrderLine extends BaseModel
+{
+    /**
+     * @var array
+     */
+    private $instructions;
+
+    /**
+     * @var int|null
+     */
+    private $price;
+
+    /**
+     * @var int|null
+     */
+    private $price_after_vat;
+
+    /**
+     * @var \MyParcelNL\Sdk\src\Model\Fulfilment\Product
+     */
+    private $product;
+
+    /**
+     * @var int|null
+     */
+    private $quantity;
+
+    /**
+     * @var string|null
+     */
+    private $uuid;
+
+    /**
+     * @var int|null
+     */
+    private $vat;
+
+    /**
+     * @param  array $data
+     */
+    public function __construct(array $data = [])
+    {
+        $this->uuid            = $data['uuid'] ?? null;
+        $this->price           = $this->intOrNull($data['price'] ?? null);
```

Comment:
Dit is meer een vraag voor de zekerheid en wegens mijn gebrek aan kennis maar kunnen (alle) bedragen niet floats zijn?

Reply:
Nee, we doen bedragen en gewichten waar mogelijk in de kleinst mogelijke eenheid, dus dat is in centen in dit geval :)